app/lib/payload-builders.js

```diff
@@ -40,7 +40,26 @@ const buildPersonUpdatePayload = (person) => ({
   secondaryTelephone: person.secondaryTelephone
 })
 
+const buildExemptionDetailsUpdatePayload = (exemptionDetails) => ({
+  indexNumber: exemptionDetails.indexNumber,
+  cdoIssued: exemptionDetails.cdoIssued,
+  cdoExpiry: exemptionDetails.cdoExpiry,
+  court: exemptionDetails.court,
+  policeForce: exemptionDetails.policeForce,
+  legislationOfficer: exemptionDetails.legislationOfficer ?? undefined,
+  insurance: {
+    company: exemptionDetails.insuranceCompany,
+    renewalDate: exemptionDetails.insuranceRenewal
+  },
+  certificateIssued: exemptionDetails.certificateIssued ?? undefined,
```

Comment:
I didn't need to set to undefined if null. Happy to leave in but didn't think it was necessary